recipe/bld.bat

```diff
@@ -80,8 +80,8 @@ mkdir build_dynamic_mdd
 cd build_dynamic_mdd
 cmake -G "%CMAKE_GENERATOR%" -D gtest_force_shared_crt=ON -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -D gtest_build_tests=ON -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" %GTEST_DIR%
 cmake --build . --target gtest_dll --config Debug
-copy Debug\gtest_dll.dll %LIBRARY_BIN%\gtest_dll-mdd.dll
-copy Debug\gtest_dll.lib %LIBRARY_LIB%\gtest_dll-mdd.lib
+copy Debug\gtest_dlld.dll %LIBRARY_BIN%\gtest_dll-mdd.dll
```

Comment:
Could the `d` suffix be an indicator that this a debug build?

Reply:
That is right: they set it here: https://github.com/google/googletest/blob/master/googletest/cmake/internal_utils.cmake#L169

I just opened this PR to trigger the CI. Adding a commit to your branch.